.codeclimate.yml

```diff
@@ -0,0 +1,8 @@
+version: "2"
+exclude_patterns:
+  - "dist/"
+  - "src/api/spec/"
+  - "src/api/test/"
+  - "docs/api/restility"
+  - "src/api/app/assets/javascripts/webui/application/cm2"
+  - "src/api/app/assets/javascripts/webui2"
```

Comment:
Not sure this is the right way around. I think we should ignore `webui` instead.

Reply:
Makes sense.